docker-compose.yml

```diff
@@ -47,38 +47,77 @@ services:
       - prometheus
     ports:
       - 3000:3000
-    network_mode: "host"
+    #network_mode: "host"
     volumes:
       - grafana_data:/var/lib/grafana
       - ./build/grafana/provisioning/:/etc/grafana/provisioning/
     env_file:
       - ./build/grafana/config.monitoring
     restart: always
 
-  #lily:
-    #container_name: lily
-    ## Select (only one) image
-    #image: filecoin/lily:v0.8.0
-    #image: filecoin/lily:v0.8.0-calibnet
-    #image: filecoin/lily:v0.8.0-calibnet-dev
-    #env_file:
-      ## Check envvars for configurable options
-      #- ./build/lily/docker.env
-    #depends_on:
-      #- prometheus
-      #- timescaledb
-      #- jaeger
-    #ports:
-      #- 1234:1234
-    #volumes:
-      #- lily_data:/var/lib/lily
-      #- ./build/lily/docker_init.sh:/usr/bin/docker_init.sh
-    #entrypoint: /usr/bin/docker_init.sh
-    #command:
-      #- daemon
+  redis-queue:
+    image: bitnami/redis:7.0
+    env_file:
+      # Check envvars for configurable options
+      - ./build/lily/redis-queue.env
+    ports:
+      - 6379:6379
+    restart: always
+
+  notifier:
+    image: filecoin/lily:v0.11.0
+    env_file:
+      # Check envvars for configurable options
+      - ./build/lily/notifier.env
+    depends_on:
+      - prometheus
+      - timescaledb
+      - jaeger
+      - redis-queue
+    ports:
+      - 1234:1234
+    volumes:
+      # holds lily datastore repo
+      - lily_notifier_data:/var/lib/lily
+      # persist params through restarts
+      - lily_notifier_tmp:/var/tmp/filecoin-proof-parameters
+      # notifier-specific config
+      - ./build/lily/notifier_config.toml:/var/lib/lily/config.toml
+      - ./build/lily/docker_init.sh:/usr/bin/docker_init.sh
+    entrypoint: /usr/bin/docker_init.sh
+    command:
+      - "daemon --bootstrap=false"
```

Comment:
Note: `--bootstrap=false` here for testing containers. May need to remove this.

Reply:
I think leaving this as is (bootstrap=false) makes sense for now 👍